lib/rucio/client/downloadclient.py

```diff
@@ -677,10 +680,6 @@ def _download_item(self, item, trace, log_prefix=''):
                         os.unlink(temp_file_path)
                         logger.warning('%sChecksum validation failed for file: %s' % (log_prefix, did_str))
                         logger.debug('Local checksum: %s, Rucio checksum: %s' % (local_checksum, rucio_checksum))
-                        try:
-                            self.client.declare_suspicious_file_replicas([pfn], reason='Corrupted')
```

Comment:
@cserf ok to take this out?

Reply:
Tobi [11:32]
can this be removed @Cedric @javor?
https://github.com/rucio/rucio/blob/master/lib/rucio/client/downloadclient.py#L681


Cedric [11:33]
Yes, it can

Reply:
Already confirmed on slack 😄